components/src/core/Drawer/Drawer.tsx

```diff
@@ -182,6 +182,7 @@ export const DrawerComponent: React.FC<DrawerComponentProps> = (props) => {
                     titleColor,
                     drawerOpen: isDrawerOpen(),
                     onItemSelect: () => {
+                        window.scrollTo(0, 0);
```

Comment:
what if the user's onClick event is not a navigation function to a new route?

Reply:
I introduced a new inheritable prop that allows the user to control if they want the window to scroll top on navigation.  The default is yes.